packages/suite-desktop/scripts/build.js

```diff
@@ -30,6 +45,7 @@ build({
     define: {
         'process.env.COMMITHASH': JSON.stringify(gitRevision),
     },
+    plugins: isDev || useMock ? [mockPlugin] : [],
```

Comment:
I think it would be clearer if we defined the `useMock` variable already reflecting the actual intention:
```js
const useMock = isDev || process.env.USE_MOCK !== undefined;
```

The usage here would then become a bit more straightforward
```suggestion
    plugins: useMock ? [mockPlugin] : [],
```

Reply:
Addressed in 4200d30.

Reply:
Thanks. I like that you also chamged it so that it's now possible to disable mock even in dev mode. 👍